subsys/mpsl/cx/Kconfig

```diff
@@ -23,13 +21,23 @@ choice MPSL_CX_CHOICE
 config MPSL_CX_THREAD
 	select NRFX_GPIOTE
 	select GPIO
-	bool "Thread Radio Coexistence"
+	bool "Thread Radio Coexistence [EXPERIMENTAL]"
 	help
 	  Radio Coexistence interface implementation according to Thread Radio
 	  Coexistence Practical recommendations for using a 3-wire PTA
 	  implementation for co-located 2.4 GHz radios.
 
-endchoice	# MPSL_CX_CHOICE
+config MPSL_CX_BT
+	depends on SOC_SERIES_NRF52X
+	select NRFX_GPIOTE
+	select GPIO
+	select NRFX_TIMER1
+	bool "Bluetooth Radio Coexistence [EXPERIMENTAL]"
```

Comment:
`select EXPERIMENTAL`

Reply:
it's not supposed to be experimental, we'll remove that